src/Sulu/Bundle/MarkupBundle/Markup/MarkupParserInterface.php

```diff
@@ -16,12 +16,29 @@
  */
 interface MarkupParserInterface
 {
+    const VALIDATE_OK = 'ok';
+    const VALIDATE_UNPUBLISHED = 'unpublished';
+    const VALIDATE_REMOVED = 'removed';
```

Comment:
Should these constants really be placed here? Isn't that specific to the `LinkTag`?